huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_account_test.go

```diff
@@ -0,0 +1,263 @@
+					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttr(resourceName, "password", "Test@123456789"),
+				),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"password"},
+			},
+		},
+	})
+}
+
+func testAccCheckRdsAccountDestroy(s *terraform.State) error {
+	c := acceptance.TestAccProvider.Meta().(*config.Config)
+	client, err := c.HcRdsV3Client(acceptance.HW_REGION_NAME)
+	if err != nil {
+		return fmt.Errorf("Error creating RDS client: %s", err)
+	}
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "huaweicloud_rds_account" {
+			continue
+		}
+
+		// Split instance_id and user from resource id
+		parts := strings.SplitN(rs.Primary.ID, "/", 2)
+		if len(parts) != 2 {
+			return fmt.Errorf("invalid id format, must be <instance_id>/<user>")
+		}
+		instanceId := parts[0]
+		userName := parts[1]
+		// items on every page, [1, 100]
+		limit := int32(100)
+		// List all db users
+		request := &model.ListDbUsersRequest{
+			InstanceId: instanceId,
+			Limit:      limit,
+			Page:       int32(1),
+		}
+
+		for {
+			response, err := client.ListDbUsers(request)
+			if err != nil {
+				return nil
+			}
+			users := *response.Users
+			if len(users) == 0 {
+				break
+			}
+			request.Page += 1
+			for _, user := range users {
+				if user.Name == userName {
+					return fmt.Errorf("Rds account still exists")
+				}
+			}
+		}
+	}
+
+	return nil
+}
+
+func testAccCheckRdsAccountExists(n string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("not found: %s", n)
+		}
+
+		if rs.Primary.ID == "" {
+			return fmt.Errorf("no ID is set")
+		}
+
+		c := acceptance.TestAccProvider.Meta().(*config.Config)
+		client, err := c.HcRdsV3Client(acceptance.HW_REGION_NAME)
+		if err != nil {
+			return fmt.Errorf("error creating RDS client: %s", err)
+		}
+
+		// Split instance_id and user from resource id
+		parts := strings.SplitN(rs.Primary.ID, "/", 2)
+		if len(parts) != 2 {
+			return fmt.Errorf("invalid id format, must be <instance_id>/<user>")
+		}
+		instanceId := parts[0]
+		userName := parts[1]
+		// items on every page, [1, 100]
+		limit := int32(100)
+		// List all db users
+		request := &model.ListDbUsersRequest{
+			InstanceId: instanceId,
+			Limit:      limit,
+			Page:       int32(1),
+		}
+
+		for {
+			response, err := client.ListDbUsers(request)
+			if err != nil {
+				return fmt.Errorf("error listing RDS db users: %s", err)
+			}
+			users := *response.Users
+			if len(users) == 0 {
+				break
+			} else {
+				request.Page += 1
+				for _, user := range users {
+					if user.Name == userName {
+						return nil
+					}
+				}
+			}
+		}
+
+		return fmt.Errorf("rds account not found")
+	}
+}
+
+func testRdsAccount_basic(rName string) string {
```

Comment:
suggest to use a base string to create huaweicloud_rds_instance

Reply:
done